src/components/articles.jsx

```diff
@@ -6,17 +6,37 @@ import Topic from "./Topic";
 
 export const Articles = () => {
   const [articles, setArticles] = useState([]);
+  const [order, setOrder] = useState("asc");
+  const [sort, setSort] = useState("created_at");
   const { topic } = useParams();
 
   useEffect(() => {
-    getArticles(topic).then((articles) => {
+    getArticles(topic, order, sort).then((articles) => {
```

Comment:
This would be far more readable/ higher quality code if you destructured data from articles on line 14, and then could just have data. articles.